### ISSUE TITLE
false DEVICE_NOT_FOUND log when simply create a new device

### ISSUE THREAD

**Issue by AlvaroVega** [telefonicaid/iotagent-node-lib#889]
Provisioning a new iota device is logging as error, and then raising and releasing an alarm.

```
iot-iota-ul                 | time=2020-09-03T10:24:20.565Z | lvl=ERROR | corr=7c1c6691-929d-4bf2-a215-9dbfad12cbc3 | trans=7c1c6691-929d-4bf2-a215-9dbfad12cbc3 | op=IoTAgentNGSI.Alarms | srv=n/a | subsrv=n/a | msg=Raising [MONGO-ALARM]: {"name":"DEVICE_NOT_FOUND","message":"No device was found with id:dispR","code":404} | comp=IoTAgent
iot-iota-ul                 | time=2020-09-03T10:24:20.569Z | lvl=ERROR | corr=7c1c6691-929d-4bf2-a215-9dbfad12cbc3 | trans=7c1c6691-929d-4bf2-a215-9dbfad12cbc3 | op=IoTAgentNGSI.Alarms | srv=n/a | subsrv=n/a | msg=Releasing [MONGO-ALARM] | comp=IoTAgent
```
But is normal not found a previous device created; it's just part of looking algorithm before register 
https://github.com/telefonicaid/iotagent-node-lib/blob/5214081f94288fc8e45092d68e991097f54e2610/lib/services/devices/deviceService.js#L1146

Produced by:
https://github.com/telefonicaid/iotagent-node-lib/blob/5214081f94288fc8e45092d68e991097f54e2610/lib/services/devices/deviceRegistryMongoDB.js#L199

Intercepted and logged by 
https://github.com/telefonicaid/iotagent-node-lib/blob/5214081f94288fc8e45092d68e991097f54e2610/lib/services/common/alarmManagement.js#L37

According with
https://github.com/telefonicaid/iotagent-node-lib/blob/5214081f94288fc8e45092d68e991097f54e2610/lib/services/devices/deviceRegistryMongoDB.js#L330

**Comment by AlvaroVega** [comment]
And the similar logs when create a new group device:

iot-iota-ul                 | time=2020-09-03T13:36:57.601Z | lvl=ERROR | corr=9627a04f-d5de-472d-8d40-95ee54187bae | trans=9627a04f-d5de-472d-8d40-95ee54187bae | op=IoTAgentNGSI.Alarms | srv=n/a | subsrv=n/a | msg=Raising [MONGO-ALARM]: {"name":"DEVICE_GROUP_NOT_FOUND","message":"Couldn\t find device group","code":404} | comp=IoTAgent
iot-iota-ul                 | time=2020-09-03T13:36:57.621Z | lvl=ERROR | corr=9627a04f-d5de-472d-8d40-95ee54187bae | trans=9627a04f-d5de-472d-8d40-95ee54187bae | op=IoTAgentNGSI.Alarms | srv=n/a | subsrv=n/a | msg=Releasing [MONGO-ALARM] | comp=IoTAgent

**Comment by fgalan** [comment]
Fixed by PR #892
